src/main/java/com/hokka1do/bookme/professionals/Professional.java

```diff
@@ -0,0 +1,34 @@
+package com.hokka1do.bookme.professionals;
+
+import org.springframework.web.bind.annotation.GetMapping;
+
+import javax.persistence.*;
+
+@Entity
+@Table
```

Comment:
What is the purpose of this annotation here?

Reply:
Purpose of this annotation is to tell spring that this column is a column in database.